### ISSUE TITLE
Force-close after executing simple MySQL query

### ISSUE THREAD

**Issue by budiony** [HeidiSQL/HeidiSQL#1194]
Nothing special in MySQL query execution, it was unsuccessful `UPDATE`, because of the foreign-key constraint failed, but force-close of application occurred, see the bug report. 
[bugreport.txt](https://github.com/HeidiSQL/HeidiSQL/files/5423266/bugreport.txt)

**Comment by ansgarbecker** [comment]
Please update to the latest build.

Was probably fixed by 6563274006c514afb3795aab4510c5c19306cb36

Relevant part of your callstack:

```
exception class    : EAccessViolation
exception message  : Access violation at address 0000000000413E26 in module 'heidisql.exe'. Read of address FFFFFFFFFFFFFFFB.

main thread ($3c0):
00413e26 heidisql.exe System                    @UStrCmp
00413e94 heidisql.exe System                    @UStrEqual
00be0696 heidisql.exe dbconnection    8650   +4 TDBObject.IsSameAs
00ee064a heidisql.exe Main            5606  +18 TMainForm.DataGridBeforePaint
00969fed heidisql.exe VirtualTrees   19406   +2 TBaseVirtualTree.DoBeforePaint
00983f1b heidisql.exe VirtualTrees   30340   +5 TBaseVirtualTree.PaintTree
009750c7 heidisql.exe VirtualTrees   23814  +23 TBaseVirtualTree.Paint
006de256 heidisql.exe Vcl.Controls              TCustomControl.PaintWindow
006d44c3 heidisql.exe Vcl.Controls              TWinControl.PaintHandler
006d542f heidisql.exe Vcl.Controls              TWinControl.WMPaint
006de1e8 heidisql.exe Vcl.Controls              TCustomControl.WMPaint
00965e0d heidisql.exe VirtualTrees   17569   +8 TBaseVirtualTree.WMPaint
0040ed32 heidisql.exe System                    TObject.Dispatch
006ccfbd heidisql.exe Vcl.Controls              TControl.WndProc
006d422b heidisql.exe Vcl.Controls              TWinControl.WndProc
00979e9c heidisql.exe VirtualTrees   25646  +32 TBaseVirtualTree.WndProc
006d33fa heidisql.exe Vcl.Controls              TWinControl.MainWndProc
00568793 heidisql.exe System.Classes            StdWndProc
7fffc5bc ntdll.dll                              KiUserCallbackDispatcher
00957060 heidisql.exe VirtualTrees   11363  +42 TVTHeader.Invalidate
0094ec62 heidisql.exe VirtualTrees    7794  +11 TVirtualTreeColumns.GetNewIndex
0094f02d heidisql.exe VirtualTrees    7905   +1 TVirtualTreeColumns.AdjustHoverColumn
009559f8 heidisql.exe VirtualTrees   10754 +303 TVTHeader.HandleMessage
00979e27 heidisql.exe VirtualTrees   25640  +26 TBaseVirtualTree.WndProc
006d33fa heidisql.exe Vcl.Controls              TWinControl.MainWndProc
00568793 heidisql.exe System.Classes            StdWndProc
7fffc494 USER32.dll                             DispatchMessageW
004efac0 heidisql.exe madExcept                 HandleException
004fbe21 heidisql.exe madExcept                 InterceptAHandleExcept
006d341e heidisql.exe Vcl.Controls              TWinControl.MainWndProc
00568793 heidisql.exe System.Classes            StdWndProc
7fffc5bc ntdll.dll                              KiUserCallbackDispatcher
7fffc494 USER32.dll                             DispatchMessageW
0088b1ee heidisql.exe Vcl.Forms                 TApplication.ProcessMessage
0088b263 heidisql.exe Vcl.Forms                 TApplication.HandleMessage
0088b6b1 heidisql.exe Vcl.Forms                 TApplication.Run
00f17b68 heidisql.exe heidisql         103  +41 initialization
7fffc3bb KERNEL32.DLL                           BaseThreadInitThunk
7fffc5b7 ntdll.dll                              RtlUserThreadStart
```

**Comment by ansgarbecker** [comment]
Most likely outdated.
